cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -155,12 +156,13 @@ public void run() {
                         dismiss();
```

Comment:
TestCase might popups the coachmark in serialised which cannot occurs in general use case. That might make the flakiness. 
I cannot say this is the problem but I feels that the removing callback might prevent the failing because that originated by this runnable.
And also think that the un-triggered runnable should be removed during the dismiss.